package.json

```diff
@@ -35,7 +35,6 @@
     "eslint": "8.56.0",
     "chai": "4.3.10",
     "mocha": "10.2.0",
-    "prettier": "3.1.0",
```

Comment:
Why is `prettier` removed?

Reply:
Really not sure, let's put it back

Reply:
oh, sorry, not, it was removed because it was a duplicate. It is already there at line 30